zulip_bots/zulip_bots/bots/jira/test_jira.py

```diff
@@ -46,6 +46,112 @@ class TestJiraBot(BotTestCase, DefaultTests):
 
     MOCK_NOTHING_RESPONSE = 'Sorry, I don\'t understand that! Send me `help` for instructions.'
 
+    MOCK_HELP_RESPONSE = '''
```

Comment:
I generally like the technique of just copying the help text into the test to validate the help command, but this help text is pretty enormous.  I'm not sure how I feel about it.  Part of the problem here is that the help text is probably way too long for human users, too, but that's not something we need to fix in this PR.  Maybe @neiljp or somebody can give a second opinion here.

Reply:
@neiljp Could you review this PR ?